python/activity_stream/widget_note.py

```diff
@@ -54,10 +54,32 @@ def __init__(self, parent):
         # self.ui.links.linkActivated.connect(self._entity_request_from_url)
         self.ui.content.linkActivated.connect(self._entity_request_from_url)
         self.ui.header_left.linkActivated.connect(self._entity_request_from_url)    
-        self.ui.user_thumb.entity_requested.connect(lambda entity_type, entity_id: self.entity_requested.emit(entity_type, entity_id))    
+        self.ui.user_thumb.entity_requested.connect(lambda entity_type, entity_id: self.entity_requested.emit(entity_type, entity_id))
+
+    ##############################################################################
+    # properties
+
+    @property
+    def user_thumb(self):
+        """
+        The user thumbnail widget.
+        """
+        return self.ui.user_thumb
 
     ##############################################################################
     # public interface
+
+    def set_user_thumb_cursor(self, cursor):
+        """
+        Sets the cursor displayed when hovering over the user
```

Comment:
This seems like an odd addition to me, it's not the kind of public interface member I would expect on a Shotgun note widget :) What's the use case here? Sorry, just curious. At the very least, it would be useful to have an example in the docstring/sphinx docs to explain when this method is useful.